app/src/test/kotlin/com/mapzen/erasermap/presenter/MainPresenterTest.kt

```diff
@@ -466,19 +468,88 @@ class MainPresenterTest {
     @Test fun onSearchResultSelected_shouldCenterOnCurrentFeature() {
         val result = Result()
         val features = ArrayList<Feature>()
+        features.add(Feature())
+        features.add(Feature())
+        val feature = SimpleFeature.create("1", "1", "", "",
+            "", "", "", "", "", "", "", 0.0, "", "", 40.0, 70.0)
+        features.add(feature.toFeature())
+        result.features = features
+        mainController.currentSearchItemPosition = 2
+        presenter.onSearchResultsAvailable(result)
+        presenter.onSearchResultSelected(0)
+        assertThat(mainController.currentSearchItemPosition).isEqualTo(2)
```

Comment:
Search result selection seems to work fine in the app but I'm not sure I follow the logic in the test.

If search result 0 is selected, why would mainController.currentSearchItemPosition be equal to 2? Shouldn't it get set to 0 on selection?